extensions/common/iam/oauth2/oauth2-core/src/test/java/org/eclipse/dataspaceconnector/iam/oauth2/core/identity/Oauth2ServiceImplTest.java

```diff
@@ -211,9 +214,9 @@ void verifyExpiredToken() {
 
     @Test
     void verifyValidJwt() {
-        var jwt = createJwt(PROVIDER_AUDIENCE, Date.from(now.minusSeconds(1000)), new Date(System.currentTimeMillis() + 1000000));
+        var jwt = createJwt(ENDPOINT_AUDIENCE, Date.from(now.minusSeconds(1000)), new Date(System.currentTimeMillis() + 1000000));
```

Comment:
Just a nit, but could we use the new endpoint audience in all test methods that verify a JWT (except for the ones testing for invalid audience of course)? Just to have it more consistent

Reply:
done